python/setup.py

```diff
@@ -272,7 +109,9 @@ def finalize_options(self):
           "Programming Language :: Python :: 3.9"
       ],
       author="NVIDIA Corporation",
-      setup_requires=['cython'],
+      url="https://github.com/rapidsai/cudf",
+      setup_requires=['Cython>=0.29,<0.30'],
+      packages=find_packages(include=['cuml', 'cuml.*']),
```

Comment:
In all the other RAPIDS packages we include the pxd files as part of the package data in case someone wants to build Cython against the Cython in our libs. Not strictly relevant here, but I thought I'd point it out in case it's an unintentional oversight. If needed the code can be copied from one of the other RAPIDS libs.